lib/Command/ExpirePassword.php

```diff
@@ -83,17 +95,28 @@ protected function configure() {
 			->setDescription('Expire a user\'s password')
 			->addArgument(
 					'uid',
-					InputArgument::REQUIRED,
+					InputArgument::OPTIONAL,
 					'The user\'s ownCloud uid (username).'
-				     )
+				)
 			->addArgument(
 					'expiredate',
 					InputArgument::OPTIONAL,
 					'The date and time when a password expires, e.g. "2019-01-01 14:00:00 CET".',
 					'-1 days'		// base.php sets timezone to utc, so
 										// make sure date is in the past
-				     )
-		;
+				)
+			->addOption(
+				'all',
+				null,
+				InputOption::VALUE_NONE,
+				'Will add password expiry to all known users.'
+			)
+			->addOption(
+				'groups',
+				'g',
+				InputArgument::OPTIONAL,
+				'Add password expiry to user(s) under group(s). This option can be used as --groups=foo,bar to scan groups foo and bar'
```

Comment:
It needs to allow for comma in a group name. There is code in ``core`` repo ``core/Command/User/Add.php`` that has a ``group``  option that can be repeated on the command line.
```
--group foo --group bar
```

Reply:
@phil-davis It does allow group names separated by comma. For example you can provide:
`./occ user:expire-password --groups=group1,group2    "2019-01-01 14:00:00 CET"`. Hope I understood your query correctly?

Reply:
My comment is about a group that contains a comma - group called ``finance,Sydney``

Reply:
@phil-davis Ouch... Thanks for pointing that... :+1:

Reply:
Done. Also updated the test to have group name with comma.